### ISSUE TITLE
TensorRT result is too different from tensorflow

### ISSUE THREAD

**Issue by lgarithm** [tensorlayer/openpose-plus#69]

**Comment by zsdonghao** [comment]
We tested on caffe version, by using TensorRT float16, the result is similar.

Btw, If the model has batch norm , please remember to set is_train to False.

**Comment by lgarithm** [comment]
This is a python script which can run tensorflow model as tensorRT directly:
https://github.com/tensorlayer/openpose-plus/blob/6bf1fc9ead91d713c1ead7087b24b48f1ba732f1/uff-runner.py
@zsdonghao @luomai need someone to help run it and compare with the result of TF.

**Comment by lgarithm** [comment]
Fixed after switching to Channels first.
